client/client.go

```diff
@@ -1247,6 +1249,27 @@ var tsoReqPool = sync.Pool{
 	},
 }
 
+// GetAllocID gets the alloc ID from PD.
+func (c *client) GetAllocID(ctx context.Context) (uint64, error) {
```

Comment:
Do we need to record the execution time?

Reply:
ok，I'll remove it